server/database/csv_writers.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+const faker = require('faker');
+
+// these need to be defined here, dont change this.
+const NUM_PRODUCTS = 1000000;
+const NUM_PHOTOS = 10000000;
+
+function productWriter() {
+  return new Promise((resolve) => {
+    const writer = fs.createWriteStream('server/database/data/products.csv');
+    writer.write('id, colorID, price, reviewscore, questions, title\n');
+    let count = 0;
+    const recursiveWrite = () => {
+      let keepWriting = true;
+      while (count < NUM_PRODUCTS && keepWriting) {
+        count += 1;
+        keepWriting = writer.write(
+          `${count},`
+          + `${faker.random.number({ min: 0, max: 1 })},`
+          + `${faker.commerce.price()},`
+          + `${faker.random.number({ min: 1, max: 5 })},`
+          + `${faker.random.number({ min: 1, max: 100 })},`
+          + `${faker.commerce.productName()}\n`,
```

Comment:
Maybe create a helper function to create this string for cleaner code. Not a big priority though.